cypress/integration/registration_validation.spec.js

```diff
@@ -173,49 +173,21 @@ describe('User opens registration form and can see validation errors', () => {
+    cy.get('[data-testid=author-radio-button]').first().click({ force: true });
     cy.get('[data-testid=connect-author-button]').click({ force: true });
     cy.contains(ErrorMessage.MISSING_AUTHOR).should('not.exist');
     cy.contains(ErrorMessage.MISSING_SUPERVISOR).should('be.visible');
 
     // Add supervisor
-    cy.get('[data-testid=add-contributor-Supervisor]').click({ force: true });
+    cy.get('[data-testid=add-contributor-Supervisor]').first().click({ force: true });
     cy.get('[data-testid=search-input]').click({ force: true }).type('test');
-    cy.get('[data-testid=author-radio-button]').eq(0).click({ force: true });
+    cy.get('[data-testid=author-radio-button]').last().click({ force: true });
     cy.get('[data-testid=connect-author-button]').click({ force: true });
     cy.contains(ErrorMessage.MISSING_AUTHOR).should('not.exist');
     cy.contains(ErrorMessage.MISSING_SUPERVISOR).should('not.exist');
 
-    // Set corresponding (and email)
-    cy.get('[data-testid=author-corresponding-checkbox]').eq(0).click({ force: true });
-    cy.contains(ErrorMessage.REQUIRED).should('not.exist');
-    cy.get('[data-testid=nav-tabpanel-reference]').click({ force: true });
-    cy.get('[data-testid=nav-tabpanel-contributors]').click({ force: true });
-    cy.contains(ErrorMessage.REQUIRED).should('be.visible');
-
-    cy.get('[data-testid=author-email-input]').click({ force: true }).type('test');
-    cy.get('[data-testid=nav-tabpanel-contributors]').click({ force: true });
-    cy.contains(ErrorMessage.INVALID_FORMAT).should('be.visible');
-    cy.get('[data-testid=author-email-input]').click({ force: true }).type('@email.com');
-    cy.get('[data-testid=nav-tabpanel-contributors]').click({ force: true });
-    cy.contains(ErrorMessage.INVALID_FORMAT).should('not.exist');
-    cy.contains(ErrorMessage.REQUIRED).should('not.exist');
-
-    // Add author and set corresponding without setting email
-    cy.get('[data-testid=add-contributor-Creator]').click({ force: true });
-    cy.get('[data-testid=search-input]').click({ force: true }).type('test');
-    cy.get('[data-testid=author-radio-button]').eq(1).click({ force: true });
-    cy.get('[data-testid=connect-author-button]').click({ force: true });
-    cy.get('[data-testid=author-corresponding-checkbox]').eq(1).click({ force: true });
-    cy.get('[data-testid=nav-tabpanel-description]').click({ force: true });
-    cy.get('[data-testid=nav-tabpanel-contributors]').click({ force: true });
-    cy.contains(ErrorMessage.REQUIRED).should('be.visible');
-    cy.get('[data-testid=nav-tabpanel-contributors]').get('[data-testid=error-tab]');
-    cy.get('[data-testid=author-email-input]').eq(1).click({ force: true }).type('test@email.com');
-    cy.get('[data-testid=nav-tabpanel-contributors]').click({ force: true });
-    cy.contains(ErrorMessage.REQUIRED).should('not.exist');
     cy.get('[data-testid=nav-tabpanel-contributors]').children('[data-testid=error-tab]').should('not.exist');
```

Comment:
TODO: tror kanskje dette er feil, og at testene vil gå gjennom om vi hadde brukt `.should('exist)` også. ser på det til en evt egen PR